packages/emmett-fastify/src/index.ts

```diff
@@ -0,0 +1,101 @@
+import Compress from '@fastify/compress';
+import Etag from '@fastify/etag';
+import Form from '@fastify/formbody';
+import closeWithGrace from 'close-with-grace';
+import Fastify, {
+  type FastifyInstance,
+  type FastifyReply,
+  type FastifyRequest,
+} from 'fastify';
+
+const defaultPlugins = [
+  { plugin: Etag, options: {} },
+  { plugin: Compress, options: { global: false } },
+  { plugin: Form, options: {} },
+];
+
+const defaultPostMiddlewares = (app: FastifyInstance) => {
+  app.all('*', async (request: FastifyRequest, reply: FastifyReply) => {
+    return reply.status(404).send();
+  });
+};
+
+export interface ApplicationOptions {
+  serverOptions?: { logger: boolean };
```

Comment:
**SUGGESTION:** I think that we should allow people to provide their own server options and just set up the defaults to make configuration smoother. I wouldn't like to wrap the whole initialisation as then each change in fastify will trigger the need to adjust the `ApplicationOptions`.